script/cluster_utilities.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ux
+
+. "${ROOT}/script/utilities.sh" || { echo 'Cannot load Bash utilities'; exit 1; }
+
+# Cleanup services in Kubernetes to prevent network resource leaking
+function wipe_cluster() {
+  local namespace
+  for namespace in $(kubectl get namespaces -oname | grep -v kube-system); do
```

Comment:
funny you should ask :) we tried it. Problem is if you just kill the cluster without wiping it clean first (and making sure the cleanup is done), GKE will leak resources which exist outside of the k8s cluster (most notably, network load balancers).